forecasts.js

```diff
@@ -18,6 +18,7 @@ var cheerio = require('cheerio');
 
 
 // avi danger levels
+forecasts.AVI_LEVEL_UNKNOWN = -1;
```

Comment:
this doesn't look right -- we overwrite this value with zero in the next line. i think we want to leave this as zero. i.e. we don't need this new line of code.

Reply:
I put it in there because avy.org is showing -1 as a danger rating. I am not sure if they will ever use 0 in place of -1 nor am I sure if there would ever be a difference. Regardless I think you are right for now it really is not needed and if someday there are two different ratings of -1 and 0 we will figure that out then. Thanks for making me think more about it.